release/test_release_gold.sh

```diff
@@ -12,6 +12,17 @@ function main {
 		return
 	fi
 
+	test_check_usage
+	test_prepare_next_release_branch_with_variable_not_set
+	test_update_release_info_date_with_variable_not_set
+
+	export LIFERAY_RELEASE_PREPARE_NEXT_RELEASE_BRANCH="no"
+
+	test_prepare_next_release_branch_with_variable_set_to_no
+	test_update_release_info_date_with_variable_set_to_no
```

Comment:
@lucasmiranda0, can you please move the content of the tests...

- `test_prepare_next_release_branch_with_variable_not_set` and `test_prepare_next_release_branch_with_variable_set_to_no` to be inside the existing method `test_not_prepare_next_release_branch`?
- `test_update_release_info_date_with_variable_not_set` and `test_update_release_info_date_with_variable_set_to_no` to be inside the existing method `test_not_update_release_info_date`?

Thanks